src/main/java/com/bookclub/controller/EventsController.java

```diff
@@ -64,6 +68,8 @@ private void selectEvent(Event event) {
         hourSpinner.getValueFactory().setValue(event.getDateTime().getHour());
         minuteSpinner.getValueFactory().setValue(event.getDateTime().getMinute());
         eventLocationField.setText(event.getLocation());
+        toggleFieldsVisibility(true);
+        toggleFieldsDisable(!event.getOrganizer().equals(LoginService.getCurrentUser().getUsername()));
```

Comment:
Can I request a TODO here to migrate to ids?
```suggestion
        // TODO: Migrate to using ids
        toggleFieldsDisable(!event.getOrganizer().equals(LoginService.getCurrentUser().getUsername()));
```